src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -63,10 +68,21 @@ public ListPanel(UI ui, GUIController guiController, PanelControl parentPanelCon
         super(ui, guiController, parentPanelControl, panelIndex);
         this.ui = ui;
         this.guiController = guiController;
-
+        totalLabel = new Label("Total : - ");
         listView = new IssueListView();
         setupListView();
         getChildren().add(listView);
+        getChildren().add(createPanelStats());
+    }
+
+    /**
+     * Create an Graphic element(HBox) containing a label that show the total no of issues in the ListPanel.
+     * @return HBox Instance to be added to the ListPanel.
+     */
+    private HBox createPanelStats() {
```

Comment:
May be call it 'PanelFooter' because we might add more than stats to that part of the panel?